frontend/lib/resolution/scope-queries.cpp

```diff
@@ -487,6 +487,11 @@ static bool doLookupInScope(Context* context,
   bool checkToplevel = (config & LOOKUP_TOPLEVEL) != 0;
   bool onlyInnermost = (config & LOOKUP_INNERMOST) != 0;
   bool skipPrivateVisibilities = (config & LOOKUP_SKIP_PRIVATE_VIS) != 0;
+  bool goPastModules = (config & LOOKUP_GO_PAST_MODULES) != 0;
+
+  // goPastModules should imply checkParents; otherwise, why would we proceed
+  // through module boundaries if we aren't traversing the scope chain?
+  CHPL_ASSERT(!goPastModules || checkParents);
```

Comment:
Seems like we should just ignore `LOOKUP_GO_PAST_MODULES` in the case where `LOOKUP_PARENTS` is not set instead of potentially crashing.

Reply:
I think, since asserts are disabled in production, leaving this assertion is fair. `LOOKUP_GO_PAST_MODULES` only applies to `LOOKUP_PARENTS`, so throwing it without `LOOKUP_PARENTS` is bad form, and I don't see a reason to do it (so why allow it?).